p2p/protocol/protocol.go

```diff
@@ -116,14 +134,15 @@ func handleRequest(quaiMsg *pb.QuaiRequestMessage, stream network.Stream, node Q
 			// TODO: handle error
 			return
 		}
+	//! Do we need to handle *common.Hash here?
```

Comment:
this question should be removed. Perhaps create an issue if there is a discovery task related to your question